MagicBytesValidator/Services/Streams/StreamFileTypeProvider.cs

```diff
@@ -29,23 +29,19 @@ CancellationToken cancellationToken
         var sequencesToFileTypes = _mapping.FileTypes
             .Select(fileType =>
                 fileType.MagicByteSequences
-                    .Select(sequence => (sequence, fileType))
+                    .Select(sequence => (length: sequence.Length + fileType.MagicByteOffset, sequence, fileType))
```

Comment:
```suggestion
                    .Select(sequence => (sequence.Length + fileType.MagicByteOffset, sequence, fileType))
```

Identifiers are not necessary

Reply:
I agree, they aren't necessary but makes for a more readable code than, say using `Item1` in line 35, but if you want me to remove it, I have no problem with it 😃